.ci/1.15/develop-with-postgres/workspace.yml

```diff
@@ -0,0 +1,20 @@
+workspace('ci-go-sample'):
+  description: generated local workspace for ci-go-sample.
+  harness: inviqa/go
+
+attributes:
+  app:
+    build: develop
+    services: [postgres]
+  go:
+    module_name: ci-go-sample
+    version: 1.15
+  services:
+    postgres:
+      version: alpine
+  database:
+    name: my_database
+    host: postgres
+    user: db-user
+    pass: db-pass
+    port: 3306
```

Comment:
Is that port right for postgres? Doesn't it default to 5432?

Reply:
wondering why CI passed🤔

Reply:
yeah that's an oversight, CI will pass fine because the app isn't actually connecting to it, and plus this attribute is telling postgres what port to bind to so it will be fine binding to port 3306 if we tell it to

Reply:
there's no config in the above link to make the docker service bind to a different port

Reply:
nor in https://github.com/inviqa/harness-go/blob/master/harness/attributes/docker.yml#L15-L22

Reply:
sorry, you are right, this attribute is just for use when setting app env vars which are not relevant here because the skeleton app does not connect to any database currently, so it's just an unused attribute currently, I will remove them